dockerfiles/alpine/micro-integrator/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk8:jdk8u212-b03-alpine
+FROM adoptopenjdk/openjdk8:jdk8u222-b10-alpine
```

Comment:
@indikasampath2000 as of now, we have used official AdoptOpenJDK Docker images as the base Docker images in Integration `v7.0.x` Docker images. In the previous Docker resource release [1], we had used the JRE based Docker image.

This has been done primarily to reduce the Docker image size.

But using the JDK based images are recommended especially when factors such as, debugging is taken into consideration.

What is the product team's preference on this?

[1]: [Product Micro Integrator Docker resources](https://github.com/wso2/micro-integrator/blob/v1.1.0/distribution/src/docker-distribution/filtered/Dockerfile#L20)